CONTRIBUTING.md

```diff
@@ -624,32 +624,36 @@ We use [Cypress](https://www.cypress.io/) for integration tests. Tests can be ru
 
 ```bash
 export SUPERSET_CONFIG=tests.superset_test_config
+export SUPERSET_TESTENV=true
+export ENABLE_REACT_CRUD_VIEWS=true
 export CYPRESS_BASE_URL="http://localhost:8081"
```

Comment:
one piece of feedback that Harrison gave on this, is that this needs to be run in the same shell window as the cypress command. Should we move this line down, or else put it on the same line, such as `CYPRESS_BASE_URL="http://localhost:8081" npm run cypress-run-chrome`

Reply:
Doesn't `export` effectively does that for the whole bash session?

Reply:
Yes, but if you open a new tab, window, or pane, it will create a new a new session.

Reply:
@eschutho true, but the same holds for the other env vars. tox, while a pain, does abstract all of the env vars away from the user.